kinematics_base/include/moveit/kinematics_base/kinematics_base.h

```diff
@@ -452,6 +513,16 @@ class KinematicsBase
   }
 
   /**
+   * @brief Sets the discretization applied to each redundant joint
+   * @param discretization a map of joint indices and discretization value pairs.  The first pair element corresponds
+   * to the index of the joint to be discretized.
```

Comment:
I meant to say that in each pair, the key is the index and the value is the discretization for the joint that corresponds to that index.